docs/README.md

```diff
@@ -6,7 +6,7 @@ If you are looking for our old [legacy API documentation, see this](https://chec
 
 ### General
 
-Our payment processing service related documentation.
+Documentation for our payment processing service.
 
 <p class="tip">
 If you have any feedback regarding how we could improve the documentation, [please file an issue on Github](https://github.com/CheckoutFinland/psp-api/issues).
```

Comment:
If you have any feedback and want us to improve the documentation, please file a issue on github.